### PR TITLE
Frontend refactor

### PR DESCRIPTION
**Pull request by zurdi15**
A lot of refactor in the front to be more maintanable and understandable. Also added some tiny details to have a better and cleaner UI.

Now Rom is better responsive in all kind of devices.

 - [x] XL and LG breakpoints logic refactor.
 - [x] Details view refactor.
 - [x] General settings view refactor.

### CODE REVIEW

File: backend/endpoints/tests/test_heartbeat.py [hunk 1]
```diff
@@ -9,13 +9,30 @@ def test_heartbeat():
...
+                "TITLE": "Scheduled Switch TitleDB update",
+                "MESSAGE": "Updates the Nintedo Switch TitleDB file",
+            },
+            "MAME_XML": {
+                "ENABLED": True,
+                "CRON": "0 5 * * *",
+                "TITLE": "Scheduled MAME XML update",
+                "MESSAGE": "Updates the Nintedo MAME XML file",
+            },
+        },
```
Comment (gantoine) [inline backend/endpoints/tests/test_heartbeat.py:37]: 👍🏼

File: frontend/src/plugins/router.js [hunk 2]
```diff
@@ -51,4 +52,9 @@ const router = createRouter({
   routes,
 });
 
+router.afterEach(() => {
+  // Scroll to top to avoid annoying behaviour in mobile
+  window.scrollTo({ top: 0, left: 0 });
```
Comment (gantoine) [inline frontend/src/plugins/router.js:57]: Hope this doesn't break anything lol

File: frontend/src/utils/utils.js [hunk 3]
```diff
@@ -28,11 +33,27 @@ export const views = {
...
 }
 
-export const defaultAvatarPath = "/assets/default_avatar.png";
+export function convertCronExperssion(expression) {
+  let convertedExpression = cronstrue.toString(expression, { verbose: true });
+  convertedExpression =
+    convertedExpression.charAt(0).toLocaleLowerCase() +
+    convertedExpression.substr(1);
+  return convertedExpression;
+}
```
Comment (gantoine) [inline frontend/src/utils/utils.js:59]: 👍🏼

**Review by GAntoine — APPROVED** [review]

File: frontend/src/plugins/router.js (hunk 2 above)
Comment (zurdi15) [reply frontend/src/plugins/router.js:57]: I tested it a lot and seems to work as intended. Crossed fingers 🤣

**Pull request merged**